models/yield_yak/arbitrum/yield_yak_arbitrum_trades.sql

```diff
@@ -0,0 +1,108 @@
+{{ config(
+    
+    alias = 'trades',
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'trace_address'],
+    post_hook='{{ expose_spells(\'["arbitrum"]\',
+                                "project",
+                                "yield_yak",
+                                \'["angus_1"]\') }}'
```

Comment:
can you remove this post hook?
we're moving away from it's usage on spells which feed into downstream spells

Reply:
Done :)